RocketChat/Rocket.Chat.ReactNative
# [FIX] Search results show URL previews

## Proposed changes
When searching for messages in a room, if the result messages have URL, the we can see URL preview.


## Issue(s)	

Closes #4061 
## How to test or reproduce

To test the changes follow the steps:
1. Open a room that contains links.
2. Go to chat search and search for messages that contain URLs.
3. We will be able to see the URL preview in the search results earlier there were blank transparent bars.
## Screenshots
<img width="245" alt="Screenshot 2022-04-15 at 1 03 31 PM" src="https://user-images.githubusercontent.com/55805094/163540006-e81f89c8-f1f9-4758-ba7a-2db2fd3e2914.png">

## Types of changes




## Checklist



## Further comments
